libs/shared/barista-definitions/src/lib/barista-definitions.ts

```diff
@@ -145,9 +145,10 @@ export interface BaStrapiPage extends BaStrapiBase {
 
 /** Strapi snippet */
 export interface BaStrapiSnippet extends BaStrapiBase {
-  slotID: string;
+  slotId: string;
```

Comment:
why do we call it `slot` once, and then `snippet` another time?

Reply:
In the beginning we talked about slots, now about snippets. I'll think about renaming it.

Reply:
I don't know. Naming things. I was thinking about having slots within the content where we then can place text/content snippets...